### PR TITLE
Config shared obj rubocop

### PR DESCRIPTION
**Pull request by desislava453**
Configurations for env and creds, added new config in rubocop and created shared_objects file with objects in it.

### CODE REVIEW

**Review by IvayloHris — APPROVED** [review]
LGTM! :)

**Review by okLetsCode — APPROVED** [review]
Good job!

File: .rubocop.yml [hunk 1]
```diff
@@ -4,9 +4,17 @@ AllCops:
...
+RSpec/InstanceVariable:
+ Enabled: false
+
 Metrics/LineLength:
-  Max: 120
\ No newline at end of file
+  Max: 200
+
+Metrics/ClassLength:
+  Max: 200
```
Comment (AleksandrosV) [inline .rubocop.yml]: I need more length 220 will be good

**Review by AleksandrosV — COMMENTED** [review]
Please add more Metrics/ClassLength:  220 will be ok

File: .rubocop.yml (hunk 1 above)
Comment (desislava453) [reply .rubocop.yml]: The requested change is done - ClassLength 220

**Review by AleksandrosV — APPROVED** [review]
Good work.

**Pull request merged**